addons/medical/scripts/Game/Damage/DamageEffects/CharacterDamageEffects/ACE_Medical_EpinephrineDamageEffect.c

```diff
@@ -0,0 +1,19 @@
+//------------------------------------------------------------------------------------------------
+class ACE_Medical_EpinephrineDamageEffect: SCR_DotDamageEffect
+{
```

Comment:
See: https://discord.com/channels/105462288051380224/976155351999201390/1244272008733134910
```suggestion
{

	//------------------------------------------------------------------------------------------------
	protected override void HandleConsequences(SCR_ExtendedDamageManagerComponent dmgManager, DamageEffectEvaluator evaluator)
	{
		super.HandleConsequences(dmgManager, evaluator);

		evaluator.HandleEffectConsequences(this, dmgManager);
	}

```